app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsController.scala

```diff
@@ -381,6 +369,54 @@ class ReliefDetailsController @Inject() (
     }
   }
 
+  private def hasRequiredPreviousAnswerForOtherReliefs(answers: ReliefDetailsAnswers): Boolean =
+    answers match {
```

Comment:
I think this is more easily read like this:
```scala
val privateResidentsRelief = answers.fold(_.privateResidentsRelief, c => Some(c.private...))
val lettingRelief =answers.fold(_.lettingRelief, c => Some(c.lettingRelief)

privateResidentRelief.exists(_.isZero) || (
  privateResidentsRelief.exists(_  > AmountInPence.zero) && lettingRelief.isDefined
)  
```
what do you think?

Reply:
Done.